tracer/test/util.go

```diff
@@ -0,0 +1,38 @@
+package test
+
+import (
+	"github.com/DataDog/dd-trace-go/tracer"
+	"net/http"
+)
+
+// GetTestTracer returns a tracer with a dummy Transport that serves
+// for contribs testing
+func GetTestTracer() (*tracer.Tracer, *DummyTransport) {
+	transport := &DummyTransport{}
+	tracer := tracer.NewTracerTransport(transport)
+	return tracer, transport
+}
+
+// DummyTransport is a transport that just buffers spans and encoding
+type DummyTransport struct {
```

Comment:
Could this use https://godoc.org/github.com/DataDog/dd-trace-go/tracer#DummyTransport instead?